platform/bukkit/plugin/src/main/java/me/lokka30/treasury/plugin/bukkit/TreasuryBukkit.java

```diff
@@ -24,6 +24,9 @@
 public class TreasuryBukkit extends JavaPlugin {
 
     private BukkitTreasuryPlugin treasuryPlugin;
+    public BukkitTreasuryPlugin getTreasuryPlugin() { return treasuryPlugin; }
```

Comment:
not needed, you are using common methods in the metrics implementation, replacing `plugin.getTreasuryPlugin()`'s usages inside `BukkitMetricsManager` with `TreasuryPlugin.getInstance()` will be fine.

Reply:
I'll run the keybind and fix the unnecessary method. Thank you :smiley:

Reply:
Will reformat code once both PRs are merged 👌🏻

Reply:
> Will reformat code once both PRs are merged 👌🏻

No. Do it now. We don't want anymore dirty commit history. I made it enough dirty due to other stuff.